NOTICE.md

```diff
@@ -17,55 +15,19 @@ SPDX-License-Identifier: Apache-2.0
-| libgcc1             | https://developer.puri.sm/licenses/Librem5/Birch/libgcc1.html                                                    |
-| libkrb5-3           | https://developer.puri.sm/licenses/Librem5/Birch/libkrb5-3.html                                                  |
-| libgssapi-krb5      | https://developer.puri.sm/licenses/Librem5/Birch/libgssapi-krb5-2.html                                           |
-| liblttng-ust0       | http://changelogs.ubuntu.com/changelogs/pool/universe/u/ust/ust_2.10.1-1/copyright                               |
-| zlib1g              | https://metadata.ftp-master.debian.org/changelogs//main/z/zlib/zlib_1.2.8.dfsg-5_copyright                       |
-| locales             | https://metadata.ftp-master.debian.org/changelogs//main/g/glibc/glibc_2.32-4_copyright                           |
-| sudo                | https://www.sudo.ws/license.html                                                                                 |
-| ncdu                | https://en.wikipedia.org/wiki/Ncdu                                                                               |
-| man-db              | https://metadata.ftp-master.debian.org/changelogs//main/m/man-db/man-db_2.9.4-2_copyright                        |
-| strace              | https://metadata.ftp-master.debian.org/changelogs//main/s/strace/strace_4.26-0.2_copyright                       |
-| init-system-helpers | https://metadata.ftp-master.debian.org/changelogs//main/i/init-system-helpers/init-system-helpers_1.60_copyright |
 | Helm                | https://github.com/helm/helm/blob/main/LICENSE                                                                   |
 | K3d                 | https://github.com/rancher/k3d/blob/main/LICENSE                                                                 |
 | Kubectl             | https://github.com/kubernetes/kubectl/blob/master/LICENSE                                                        |
-| Brew                | https://github.com/Homebrew/brew/blob/master/LICENSE.txt                                                         |
 | K9S                 | https://github.com/derailed/k9s/blob/master/LICENSE                                                              |
 | Redis               | https://redis.com/legal/licenses/                                                                                |
 | Mosquitto           | https://github.com/eclipse/mosquitto/blob/master/LICENSE.txt                                                                                                                 |
 
 ### Development
 | Software |                               License                                |
 | :------: | :------------------------------------------------------------------: |
-|  Thread  | https://github.com/loretoparisi/python-threading/blob/master/LICENSE |
 |   GRPC   |           https://github.com/grpc/grpc/blob/master/LICENSE           |
```

Comment:
Is grpc and grpcio the same things?

Reply:
I would say GRPC is the general idea of communication and grpcio is one implementation of it.

Reply:
From the license point of view - they are the same things